hip-service/Link/Patient/ErrorMessage.cs

```diff
@@ -0,0 +1,11 @@
+namespace hip_service.Link.Patient
+{
+    public static class ErrorMessage
+    {
+        public const string NoLinkRequestFound = "No request found";
+        public const string CareContextNotFound = "Care Context Not Found";
+        public const string NoPatientFound = "No patient Found";
+        public const string DatabaseStorageError = "Unable to store data to Database";
+        public const string InternalServerError = "Internal Server Error";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this constant to a 'static' read-only property.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4807294)